### PR TITLE
Added Join Organization Issue Template

### LINKED ISSUE #4: Issue Templates
We should update the issue templates on this repository. Since it'll be a community resource, and not a code project, we can include issue templates for things like getting support, joining the organisation, and similar events.

### PR DESCRIPTION
**Pull request by prathamesh-borse**
# Pull Request
This pull request closes an issue related to the get involved beginners who wish to contribute to this community.

## Description:
This pull request is made for adding the issue template that is related to adding newcomers or beginners to the community.

## Related Issue:

Related #4

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Join Organization
+description: This issue template can be helpful for beginner who wish to join the community..
+title: "Invite me to the organization"
+labels: ["🚦 status: awaiting triage"]
+assignees: prathamesh-borse
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/join_org.yml:5]:
```suggestion
assignees: nhcarrigan
```
This should be me 🙂

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - type: textarea
+    id: joinorgdescription
+    attributes:
+      label: Please Explain why do you want to join the organization? 
+      description: |
+        Please provide a clear description that why do you want to join this organization.
+        
+        Include the steps you took to cause the bug, and an explanation of what occurred.
+    validations:
+      required: true
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/join_org.yml:19]:
```suggestion
      required: true

```
Need a new line at the end of the file.

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: Join Organization
+description: This issue template can be helpful for beginner who wish to join the community..
+title: "Invite me to the organization"
+labels: ["🚦 status: awaiting triage"]
+assignees: prathamesh-borse
+body:
+  - type: markdown
+    attributes:
+      value: '# Join Organization'
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/join_org.yml:9]:
```suggestion
      value: '# Join Our Organization'
```
Just a nitpick.

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  - type: textarea
+    id: joinorgdescription
+    attributes:
+      label: Please Explain why do you want to join the organization?
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/join_org.yml:13]:
```suggestion
      label: Why do you want to join our organization? 
```

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    attributes:
+      value: '# Join Organization'
+  - type: textarea
+    id: joinorgdescription
+    attributes:
+      label: Please Explain why do you want to join the organization? 
+      description: |
+        Please provide a clear description that why do you want to join this organization.
+        
+        Include the steps you took to cause the bug, and an explanation of what occurred.
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/join_org.yml:17]:
```suggestion
      description: Please provide a clear description of why you want to join our organization.
```

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+name: Join Organization
+description: This issue template can be helpful for beginner who wish to join the community..
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/join_org.yml:2]:
```suggestion
description: Use this template to request an invitation to our organization.
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
This is looking good. Just some small tweaks.

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+title: "Invite me to the organization"
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/join_org.yml:3]: This is just an idea; I'm not sure if it's good, though.

```suggestion
title: "[ORG] Invite me to the organization"
```

File: .github/ISSUE_TEMPLATE/join_org.yml [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+labels: ["🚦 status: awaiting triage"]
+assignees: prathamesh-borse
+body:
+  - type: markdown
+    attributes:
+      value: '# Join Organization'
+  - type: textarea
+    id: joinorgdescription
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/join_org.yml:11]:
```suggestion
    id: description
```

**Review by nhcarrigan — APPROVED** [review]
Looks good to me.

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Comment by prathamesh-borse** [comment]
@nhcarrigan can I Delete this branch now?

**Comment by prathamesh-borse** [comment]
and also @nhcarrigan we can close #4 this Now what do you say?

**Comment by nhcarrigan** [comment]
I intentionally left the issue open so we can continue discussing other potential templates.

**Comment by prathamesh-borse** [comment]
> I intentionally left the issue open so we can continue discussing other potential templates.

@nhcarrigan can I delete the issue template branch what do you say?

**Comment by Panquesito7** [comment]
> can I delete this branch now?

Yes, you can delete the branch safely now. 🙂

**Comment by prathamesh-borse** [comment]
> > can I delete this branch now?
> 
> Yes, you can delete the branch safely now. 🙂

@Panquesito7 Thanks for Replying! 👍

**Pull request merged**